lib/assessment.js

```diff
@@ -13,14 +21,23 @@ var min = function min() {
 // and returns the sum of that set
 // assume that all the argument are numbers
 var sum = function sum() {
-
+  var results = [];
+  for (var i = 0; i < results.length; i++) {
+    total += results[i]
+  }
+  return total;
 };
 
 // Question 3
 // Write a function that returns an array of the specified length
 // with all the elements set to defaultValue
+// COME BACK
```

Comment:
In case it wasn't clear, in this example, `defaultValue` is a single value, e.g. `"a"`; calling `newArray(5,"a")` would give us back the array `["a","a","a","a","a"].